docs/Manual/Deployment/Kubernetes/DeploymentResource.md

```diff
@@ -443,3 +432,21 @@ For more information on tolerations, consult the [Kubernetes documentation](http
 This setting specifies a set of labels to be used as `nodeSelector` for Pods of this node.
 
 For more information on node selectors, consult the [Kubernetes documentation](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/).
+
+## Deprecated Fields
+
+### `spec.<group>.resources.requests.storage: storageUnit`
+
+This setting specifies the amount of storage required for each server of this group.
+The default value is `8Gi`.
+
+This setting is not available for group `coordinators`, `syncmasters` & `syncworkers`
+because servers in these groups do not need persistent storage.
+
```

Comment:
Add: Please use VolumeClaimTemplate from now on. This field is not considered if VolumeClaimTemplate is set. Note however, that the information in requests is completely handed over to the pod in this case.